src/appleseed/renderer/kernel/aov/aovaccumulator.cpp

```diff
@@ -195,8 +195,8 @@ AOVAccumulatorContainer::AOVAccumulatorContainer()
 }
 
 AOVAccumulatorContainer::AOVAccumulatorContainer(const AOVContainer& aovs)
-  : AOVAccumulatorContainer()
 {
+    AOVAccumulatorContainer();
```

Comment:
This is not calling the default constructor. It's actually creating a new unnamed AOVAccumulatorContainer.